### PR TITLE
Elasticsearch health source - new UI

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by harshilbits — APPROVED** [review]

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 1]
```diff
@@ -1,174 +1,214 @@
...
-This option is available only if you have configured the service and environment as fixed values.
+To add a health source:
 
-A Health Source is basically a mapping of a Harness Service to the service in a deployment environment monitored by an APM or logging tool.
+1. In the **Health Sources** section, select **+ Add New Health Source**.
+   The Add New Health Source dialog appears.
+2. On the **Define Health Source** tab, do the following:
+      1. In the **Define Health Source** section, select **ElasticSearch** as health source type.
+      2. In the **Health Source Name** field, enter a name for the health source.
+      3. In the **Connect Health Source** section, select the **Select Connector**.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:122]: no need for "the" before **Select Connector**.

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 2]
```diff
@@ -1,174 +1,214 @@
...
-8. Click **Next**.  
-The **Customize Health Source** settings appear.  
-You can customize the metrics to map the Harness Service to the monitored environment in **Query Specifications and Mapping** settings.Enter a name for the query in **Name your Query**.The subsequent settings in **Customize Health Source** depend on the Health Source Type you selected. Click **Map Queries to Harness Services** drop down
-1. Click **Add Metric**.
-2.  Enter a name for the query in **Name your Query**.
-3.  Click **Fetch Records** to retrieve the details. The results are displayed under **Records.**![](./static/verify-deployments-with-elastic-search-105.png)
-4.  Once the records are fetched, click the plus icon in **Identify Service Instance** to select the path for service instance.
-5.  Click **Submit**. The Health Source is displayed in the Verify step.![](./static/verify-deployments-with-elastic-search-106.png)
+:::info note
+Currently, Harness supports only Elasticsearch logs. The **ElasticSearch Logs** option is selected by default in the **Select Feature**.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:131]: Not sure what this means: "...in the **Select Feature**."

**Review by StrawberryPad — APPROVED** [review]
Hi Sudhi, looks good! Some suggestions below that would have a good impact on this content.

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 3]
```diff
@@ -1,174 +1,214 @@
...
+1. Select the **Enable Verification** option.  
+   The Verify step gets added to the pipeline.
+2. Select the **Verify** step.  
+   The Verify settings page appears.
 
-1. In **Name**, enter a name for the step.
-2. In **Timeout**, enter a timeout value for the step.
 
-You can use:
+### To an existing deployment stage
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:47]: Updated

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 4]
```diff
@@ -1,174 +1,214 @@
...
-## Step 1: Add Verify Step
+To add a Verify step to your pipeline, use one of the following methods:
 
-There are two ways to add the Verify step:
 
-* **When selecting the stage deployment strategy:**  
-The **Verify** step can be enabled in a CD stage the first time you open the **Execution** settings and select the deployment strategy. When you select the deployment strategy you want to use, there is also an **Enable Verification** option. Select the **Enable Verification** option.  
-Harness will automatically add the **Verify** step. For example, here is a stage where Canary strategy and the **Enable Verification** option were selected.[![](./static/verify-deployments-with-elastic-search-97.png)](./static/verify-deployments-with-elastic-search-97.png)
-* **Add the Verify step to an existing Execution setup:** You can also add the Verify step to the Execution section of a CD stage in a Pipeline you previously created. Simply click **Add Step** after the deployment step, and then select **Verify**.[![](./static/verify-deployments-with-elastic-search-99.png)](./static/verify-deployments-with-elastic-search-99.png)
+### While building a deployment stage
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:37]: Updated

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 5]
```diff
@@ -1,174 +1,214 @@
...
-You can set up a Monitored Service in the Service Reliability Management module or in the Verify step in a CD stage. The process is the same.
+To set up CV, you need to configure a Service Reliability Management (SRM)-monitored service. A monitored service is a mapping of a Harness service to a service that is being monitored by your Application Performance Monitoring (APM) or logging tool.
 
-No matter where you set up the Monitored Service, once it's set up, it's available to both Service Reliability Management and CD modules.
 
-In this topic we'll set up the Monitored Service as part of the **Verify** step.
+## Add Verify Step
 
-## Step 1: Add Verify Step
+To add a Verify step to your pipeline, use one of the following methods:
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:34]: Updated

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 6]
```diff
@@ -1,174 +1,214 @@
...
-* **Add the Verify step to an existing Execution setup:** You can also add the Verify step to the Execution section of a CD stage in a Pipeline you previously created. Simply click **Add Step** after the deployment step, and then select **Verify**.[![](./static/verify-deployments-with-elastic-search-99.png)](./static/verify-deployments-with-elastic-search-99.png)
+### While building a deployment stage
 
+If you're building a deployment stage and currently on the **Execution Strategies** page:
 
-## Step 2: Enter a Name and Timeout
+1. Select the **Enable Verification** option.  
+   The Verify step gets added to the pipeline.
+2. Select the **Verify** step.  
+   The Verify settings page appears.
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:44]: Done

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 7]
```diff
@@ -1,174 +1,214 @@
...
-* `m` for minutes
-* `s` for seconds
-* `ms` for milliseconds
+If you already have a deployment stage:
 
-The maximum is `53w`.Timeouts can be set at the Pipeline level also.
+1. Select the stage where you want to add the Verify step.
+2. On the stage settings pane, select the **Execution** tab.
+3. On the pipeline, hover over where you want the Verify step, select the + icon, and then choose **Add Step**.  
+The Step Library page appears.
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:54]: Added blank lines.

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 8]
```diff
@@ -1,174 +1,214 @@
...
+
+3. The maximum timeout value you can set is **53w**. You can also set timeouts at the pipeline level.
+ 
+
+## Select a continuous verification type, sensitivity, and duration
+
+1. In **Continuous Verification Type**, select a type that matches your deployment strategy. The following options are available:
+   
+   - **Auto**: Harness automatically selects the best continuous verification type based on the deployment strategy.
+   - **Rolling Update**: A rolling deployment is a deployment technique that gradually replaces old versions of a service with a new version by replacing the infrastructure on which the service runs. Rolling updates are useful in situations where a sudden changeover might cause downtime or errors.
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:79]: Went with option B :)

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md [hunk 9]
```diff
@@ -1,174 +1,214 @@
...
+4. In the **Artifact Tag** field, reference the primary artifact that you added in the **Artifacts** section of the Service tab. Use the Harness expression `<+serviceConfig.artifacts.primary.tag>` to reference this primary artifact. To learn about artifact expression, go to [Artifact](https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/#artifact).
+5. Select **Fail On No Analysis** if you want the pipeline to fail if there is no data from the health source. This ensures that the deployment fails when there is no data for Harness to analyze.
 
-In **Monitored Service**, click **Click to autocreate a monitored service**.
+
+## Create a monitored service
+
+In **Monitored Service**, select **Click to autocreate a monitored service**.
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:92]: Updated

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md (hunk 1 above)
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:122]: Removed

File: docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md (hunk 2 above)
Comment (SudheendraKatte) [reply docs/continuous-delivery/cd-execution/cv-category/verify-deployments-with-elastic-search.md:131]: Corrected - "...in the Select Feature field."

**Review by StrawberryPad — APPROVED** [review]

**Pull request merged**